tests/mocked/test_droplets.py

```diff
@@ -610,7 +610,7 @@ def test_list_snapshots(mock_client: Client, mock_client_url):
 
 @responses.activate
 def list_supported_backup_policies(mock_client: Client, mock_client_url):
-    """Mocks the supported backup policies."""
+    """Mocks the list supported backup policies."""
```

Comment:
Nice catch :+1: Let's add 'of'?
```suggestion
    """Mocks the list of supported backup policies."""
```